x-pack/plugins/lens/public/datatable_visualization/components/columns.tsx

```diff
@@ -173,6 +173,16 @@ export const createGridColumns = (
                 'data-test-subj': 'lensDatatableResetWidth',
                 isDisabled: initialWidth == null,
               },
+              {
+                color: 'text',
+                size: 'xs',
+                onClick: () => onColumnHide({ columnId: field }),
```

Comment:
It seems like this does not update the visibility indicators in the main menu: when I use the `Hide` button from the column menu, not from the visualization menu, I get inconsistent results:

<img width="876" alt="Screen Shot 2021-01-29 at 5 41 40 PM" src="https://user-images.githubusercontent.com/666475/106334759-5144b200-6259-11eb-8ea7-03266e397381.png">